docs/index.md

```diff
@@ -15,46 +15,50 @@ In order to integrate with Lead Safe, the health provider will need to make a RE
+| race            | Text     | Code Values from Centricity        | Standard ONC Race definitions <br> <br> **Code / description** <br> 1002-5 / American Indian or Alaska Native <br> 2028-9 / Asian <br> 2054-5 / Black or African American <br> 2076-8 / Native Hawaiian or Other Pacific Islander <br> 2106-3 / White <br> UNK / Unknown |
+| ethnicity       | Text     | Code Values from Centricity        | Standard ONC Ethnicity definitions <br> <br> **Code / description** <br> 2135-2 / Hispanic or Latino <br> 2186-5 / Non Hispanic or Latino <br> UNK / Unknown |
+| VISIT ARRAY     |          |                                    |                                                                                                                   |
+| visit.visit_id  | Numeric  | 16 Digit GE ID                     | Unique GE ID for specific visit (DocumentID)                                                                      |
+| visit.date      | Date     | yyyy-mm-dd hh:mm:ss.sss-hh:mm      | Date of visit (RFC 3339 complient date)                                                                           |
+| visit.location  | Text     | <N/A>                              | Text Abbreviation for facility of visit location                                                                  |
+| visit.provider  | Text     | <N/A>                              | Full provider name, last name, or just NPI                                                                        |
+| LAB ARRAY       |          |                                    |                                                                                                                   |
+| lab.lab_id      | Numeric  | 16 Digit GE ID                     | Unique GE ID for specific lab result (ObsID)                                                                      |
+| lab.type        | Text     | "BLL"                              | Static BLL unless we identify additional labs to include                                                          |
+| lab.date        | Date     | yyyy-mm-dd                         | Date of lab results                                                                                               |
+| lab.sample_type | Text     | V/C                                | "V" Venous or "C" for Capillary                                                                                   |
+| lab.result      | Text     | <N/A>                              | Several values are possible; integer, non-integer numeric, ranges indicated by alphanumeric text.                 |
 
 ```json
 {
-  "timestamp": "2017-08-22 12:00:00.000000", 
+  "timestamp": "2017-08-22 12:00:00.000-00:00", 
+  "visit_date": "2017-08-22",
+  "network_id": "Alliance Health",
   "clinic_id": "EF",
   "location_id": "examp_loc",
-  "alliance_id": 9000, 
+  "patient_id": 9000, 
```

Comment:
In the spec above `patient_id` is Text, but here is it numeric. I think Text is right (in order to support other EHRs that use text ids) so this should be "9000".

Reply:
Agreed. Likewise, we saw that `patient_id` in the response was also not in quotes. Great catch.